erpnext/payroll/doctype/employee_benefit_claim/employee_benefit_claim.js

```diff
@@ -12,5 +12,24 @@ frappe.ui.form.on('Employee Benefit Claim', {
 	},
 	employee: function(frm) {
 		frm.set_value("earning_component", null);
+		if (frm.doc.employee) {
+			frappe.call({
+				method: "erpnext.payroll.doctype.salary_structure_assignment.salary_structure_assignment.get_employee_currency",
```

Comment:
set currency dynamically. for amount field.
![image](https://user-images.githubusercontent.com/32095923/99652688-26322b00-2a7e-11eb-8731-1e3c22816dc1.png)

Reply:
It doesn't show up until it is saved or there is another currency field and you start filling it.